frontend-test.sh

```diff
@@ -19,3 +19,7 @@ fi
 docker-compose -f docker/docker-compose.test.yml \
                -p listenbrainz_test \
                run --rm frontend_tester npm test
+
+docker-compose -f docker/docker-compose.test.yml \
```

Comment:
why did you add this back? We removed it in #790. If we want to keep it, I think frontend.sh should have options that allow for running just the tests or just the type checker.

Reply:
I added it so that, type checking can be done on all files once before commit rather than opening each file and checking for errors. I have added an option to run type checker only, using `./frontend-test.sh -t`.